packages/fxa-settings/src/components/Settings/FlowRecoveryKeyInfo/index.tsx

```diff
@@ -0,0 +1,110 @@
+import React from 'react';
+import { FlowContainer } from '../FlowContainer';
+import { ProgressBar } from '../ProgressBar';
+import { SecurityShieldImage } from '../../images';
+import { ShieldIconListItem, KeyIconListItem } from '../../IconListItem';
+import { FtlMsg } from 'fxa-react/lib/utils';
+import { useFtlMsgResolver } from '../../../models';
+import { logViewEvent, usePageViewEvent } from '../../../lib/metrics';
+
+export type FlowRecoveryKeyInfoProps = {
+  navigateForward: () => void;
+  navigateBackward: () => void;
+};
+
+export const viewName = 'settings.account-recovery.info';
+export const backwardNavigationEventName = 'navigate-forward';
+export const forwardNavigationEventName = 'navigate-backward';
+
+export const FlowRecoveryKeyInfo = ({
+  navigateForward,
+  navigateBackward,
+}: FlowRecoveryKeyInfoProps) => {
+  usePageViewEvent(viewName);
+  const ftlMsgResolver = useFtlMsgResolver();
+
+  const localizedContainerPageTitle = ftlMsgResolver.getMsg(
+    'flow-recovery-key-info-page-title',
+    'Account Recovery Key'
+  );
+
+  return (
+    <FlowContainer
+      title={localizedContainerPageTitle}
+      customBackButtonTitle="Settings"
```

Comment:
Same as above, should be "Back to settings", and perhaps pass in as `localizedCustomBackButtonTitle`?